certified-connectors/Yeeflow/apiDefinition.swagger.json

```diff
@@ -1126,6 +1371,15 @@
         }
       }
     },
+    "itemID": {
+      "name": "id",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "ListDataID",
```

Comment:
@lulut Can you please valid description value as description values should be free from grammatical and spelling errors.

Documentation - [Descriptions of properties](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)